prow-configurations/config.yaml.tpl

```diff
@@ -38,6 +38,17 @@ presubmits: # runs on PRs
     spec:
       containers:
       - image: eu.gcr.io/kyma-project/snapshot/test/integration:0.0.1 # created by running `docker build -t <image> .` in the integration-job directory.
+  - name: kyma-gke-integration
+    trigger: "(?m)^/test kyma-gke-integration"
+    rerun_command: "/test kyma-gke-integration"
+    context: kyma-gke-integration
+    skip_report: false # from documentation: SkipReport skips commenting and setting status on GitHub.
+    max_concurrency: 10
```

Comment:
specify run_if_changed param. Please review other params that are set in `kyma-integration` job